CMakeLists.txt

```diff
@@ -35,6 +35,17 @@ option(BENCHMARK_DOWNLOAD_DEPENDENCIES "Allow the downloading and in-tree buildi
 option(BENCHMARK_ENABLE_GTEST_TESTS "Enable building the unit tests which depend on gtest" ON)
 
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+if(MSVC)
```

Comment:
Please can you add a link to the upstream bugfix/bugreport?

Reply:
Is the link provided below sufficient or do you want something else as well?